website/content/api-docs/agent/check.mdx

```diff
@@ -245,6 +245,8 @@ The check sends datagrams to the value specified at the interval specified in th
 If the datagram is sent successfully or a timeout is returned, the check is set to the `passing` state.
 The check is logged as `critical` if the datagram is sent unsuccessfully.
 
+- `OSService` `(string: "")` - Specifies an OS-level service to check. You can specify either `Windows Services` on Windows or `SystemD` services on Unix.
```

Comment:
```suggestion
- `OSService` `(string: "")` - Specifies the type of OS-level service to check. You can specify either `Windows Services` on Windows or `SystemD` services on Unix.
```
Apologies, I think my last suggestion was still too ambiguous. I think this makes the purpose of the parameter clearer.

Reply:
It's not the type that we have to specify but its name

Reply:
Also, this is related: https://github.com/hashicorp/consul/pull/13388#discussion_r977810915

How about: 
```suggestion
- `OSService` `(string: "")` - Specifies the name/identifier of an OS-level service to check. You can specify either `Windows Services` on Windows or `SystemD` services on Unix.
```

Reply:
Hey @trujillo-adam ! Any thoughts on this?

Reply:
@trujillo-adam: sounds good!

```suggestion
- `OSService` `(string: "")` - Specifies the identifier of an OS-level service to check. You can specify either `Windows Services` on Windows or `SystemD` services on Unix.
```